src/coreclr/vm/interpexec.cpp

```diff
@@ -2256,6 +2256,11 @@ void InterpExecMethod(InterpreterFrame *pInterpreterFrame, InterpMethodContextFr
                     break;
                 }
 
+                case INTOP_CALL_NULLCHECK:
+                    // Check the target this pointer in the call for null
+                    NULL_CHECK(LOCAL_VAR(ip[2], void*));
+                __fallthrough;
+                    // fall through
```

Comment:
```suggestion
                    FALLTHROUGH;
```
This is the most common pattern in the VM